micronaut/README.md

```diff
@@ -15,23 +15,8 @@ Make sure the app name used here matches with the `STACK_NAME` present under `lo
 
 ### Option 2: GraalVM Native Image
 
-On MacOS:
 ```bash
-docker run --mount type=bind,source=$(pwd),destination=/project -it --entrypoint /bin/bash marksailes/al2-graalvm:11-22.0.0.2
-```
-
-On Windows:
-```bash
-docker run -v <MICRONAUT_DIR_ABSOLUTE_PATH>:/project -it --entrypoint /bin/bash marksailes/al2-graalvm:11-22.0.0.2
-```
-Make sure to replace `MICRONAUT_DIR_ABSOLUTE_PATH` with absolute path to micronaut directory.
- 
-Once docker downloads the image and runs, you would see a bash command that will run inside docker container.
-Run below command:
-
-```bash
-mvn clean package -Dpackaging=native-image -Pnative
-exit
+mvn clean package -Dpackaging=docker-native -Dmicronaut.runtime=lambda -Pgraalvm
```

Comment:
Great simplification!

Reply:
Verified that it works on windows

Reply:
Sorry, spoke a little too soon. On windows, it needed a small change in the command line option. Still much simpler than before. This is what worked:
mvn clean package -Dpackaging=docker-native -Dmicronaut_runtime=lambda -Pgraalvm

Reply:
Sorry. Spoke a little too soon. Seems like there is a slight difference with cmd vs powershell in windows. Above command works with cmd. However, this is what worked in powershell:
`mvn clean package -Dpackaging=docker-native -Dmicronaut_runtime=lambda -Pgraalvm`
So, if underscore works on mac, we should change the above command.